packages/formation/js/accordion.js

```diff
@@ -67,65 +74,65 @@ const getOtherButtons = (element, target) =>
   );
 
 const addAccordionClickHandler = () => {
-  document
-    .querySelectorAll('.usa-accordion, .usa-accordion-bordered')
-    .forEach(element => {
-      const parents = getParents(element, '.usa-accordion');
-
-      if (parents.length === 0) {
-        element.addEventListener('click', e => {
-          const accordionButton = e.target.closest('.usa-accordion-button');
-
-          // Checks whether the button has a click event already assigned to it.
-          // and if it is a .usa-accordion-button.
-          // Specifically React Components.
-          if (accordionButton && !accordionButton.onclick) {
-            const multiSelectable = toBoolean(
-              element.getAttribute('aria-multiselectable'),
-            );
+  Array.from(
```

Comment:
The diff went crazy here, I just added an `Array.from` and formatted with Prettier.